myia/analyze/value.py

```diff
@@ -0,0 +1,131 @@
+"""Value inference."""
+from typing import Dict, Callable
+from .graph import Plugin
+
+from myia import primops as P
+from myia.primops import Primitive
+from myia.utils import Named
+from myia.anf_ir import Constant, Apply, Parameter, Graph
+from myia.py_implementations import implementations as py_implementations
+
+ESTIMATORS: Dict[Primitive, Callable] = dict()
+
+
+NO_VALUE = Named('NO_VALUE')
+NOT_CONSTANT = Named('NOT_CONSTANT')
```

Comment:
In what circumstances would a node be associated to the `NOT_CONSTANT` value? From a quick grep of the source I don't see it instantiated anywhere, and it's not tested either.

Reply:
Currently none, you are right.  I should probably just remove it and we can add it back later if we improve the value inference.